packages/jwt-verifier/README.md

```diff
@@ -17,10 +17,10 @@ To learn more about verification cases and Okta's tokens please read [Working Wi
 
 ## How to use
 
-Install this library from npm:
+We use Yarn as our node package manager. To install Yarn, check out their [install documentation](https://yarnpkg.com/en/docs/install).
```

Comment:
I'm not sure we want to expose yarn at this top-level - a consumer of the package can use yarn or npm to install (usually, you'll see documentation that just uses npm or shows commands for installs with either npm or yarn). Yarn only comes into play when you're talking about development, which could be summed up in one section in the root README (or linked from the root README).